### PR TITLE
feat: Add environment about Parallel number limit of artifact-unzip-service

### PR DESCRIPTION
**Pull request by sugarnaoming**
## Context
I have added a new environment variable `WORKER_PARALLEL_UPLOAD_LIMIT` to artifact-unzip-service and will add a description of it.
Details of added features
- https://github.com/screwdriver-cd/artifacts-unzip-service/pull/7
- https://github.com/screwdriver-cd/artifacts-unzip-service/pull/9

## Objective
- Add environment (English)
- Add environment (Japanese)

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: docs/cluster-management/configure-artifacts-unzip-service.md [hunk 1]
```diff
@@ -74,6 +74,7 @@ WORKER_MIN_TASK_PROCESSORS | 1 | The minimum number of workers to spawn under th
 WORKER_MAX_TASK_PROCESSORS | 10 | The maximum number of workers to spawn under this multiWorker.
 WORKER_CHECK_TIMEOUT | 1000 | How often to check if the event loop is blocked (in ms)
 WORKER_MAX_EVENT_LOOP_DELAY | 10 | How long the event loop has to be delayed before considering it blocked (in ms)
+WORKER_PARALLEL_UPLOAD_LIMIT| 0 | Number of parallel executions of decompressing ZIP files and uploading to the Store (unlimited when 0)
```
Comment (yuichi10) [inline docs/cluster-management/configure-artifacts-unzip-service.md]: I think technically we do not decompress Zip file. How about just  "Number of parallel executions of uploading to the Store"

Also "unlimited when less than 0" is more precisely than "unlimited when 0".
https://github.com/screwdriver-cd/artifacts-unzip-service/blob/main/lib/jobs.js#L55

Comment (sugarnaoming) [reply docs/cluster-management/configure-artifacts-unzip-service.md]: thx.
I fixed it.

**Review by yoshwata — APPROVED** [review]

**Review by yuichi10 — APPROVED** [review]

File: docs/cluster-management/configure-artifacts-unzip-service.md [hunk 2]
```diff
@@ -74,6 +74,7 @@ WORKER_MIN_TASK_PROCESSORS | 1 | The minimum number of workers to spawn under th
 WORKER_MAX_TASK_PROCESSORS | 10 | The maximum number of workers to spawn under this multiWorker.
 WORKER_CHECK_TIMEOUT | 1000 | How often to check if the event loop is blocked (in ms)
 WORKER_MAX_EVENT_LOOP_DELAY | 10 | How long the event loop has to be delayed before considering it blocked (in ms)
+WORKER_PARALLEL_UPLOAD_LIMIT| 0 | Number of parallel executions of uploading to the Store (unlimited when less than 0)
```
Comment (itleigns) [inline docs/cluster-management/configure-artifacts-unzip-service.md]: This means the maximum number?

Comment (sugarnaoming) [reply docs/cluster-management/configure-artifacts-unzip-service.md]: thx.
I fixed it.

**Review by itleigns — APPROVED** [review]

**Review by tkyi — APPROVED** [review]

**Pull request merged**